scene/resources/particle_process_material.cpp

```diff
@@ -720,7 +720,7 @@ void ParticleProcessMaterial::_update_shader() {
 			code += "	local_pos.z = 0.;\n";
 			code += "	mat3 z_rotation_mat = mat3(\n";
 			code += "		vec3(cos(orbit_velocities.z),-sin(orbit_velocities.z),0.0),\n";
-			code += "		vec3(-sin(orbit_velocities.z),cos(orbit_velocities.z), 0.0),\n";
+			code += "		vec3(sin(orbit_velocities.z),cos(orbit_velocities.z), 0.0),\n";
```

Comment:
As far as I can tell, this code is working with columns. Therefore this should be the other way:

```
			code += "		vec3(cos(orbit_velocities.z), sin(orbit_velocities.z), 0.0),\n";
			code += "		vec3(-sin(orbit_velocities.z), cos(orbit_velocities.z), 0.0),\n";
```

(or, if this was rows, then the other two cases need to be swapped)

Reply:
I also checked this and got into the same conclusion, it should be changed as suggested in the above comment.

`z_rotation_mat` as currently in this PR would rotate according to the left-hand-rule, contrary to `x_rotation_mat`/`y_rotation_mat` which both rotate according to the right-hand-rule (which is correct as Godot uses right handed 3D coordinate system).
[Left and Right handed Coordinate Systems with rotations](https://coordinatesystemforgames.blogspot.com/)

Reply:
@kleonc Actually the rotation matrices are the same in both left-hand and right-hand. If you mirror the axes (which switches the handedness), and you also switch hands, the rotation direction is the same.

![hands](https://github.com/godotengine/godot/assets/1646875/6fd9cd4f-87b8-473f-8a36-c7fc9bd31073)

Reply:
> @kleonc Actually the rotation matrices are the same in both left-hand and right-hand. If you mirror the axes (which switches the handedness), and you also switch hands, the rotation direction is the same.

@aaronfranke I think you've misunderstood what I wrote. Yes, right-hand-rule-rotation in right handed coordinate system and left-hand-rule-rotation in left handed coordinate system both have the same rotation matrix. But left-hand-rule-rotation in right handed coordinate system does not, it has a mirrored matrix, just like the `z_rotation_mat` previously in this PR (it would rotate in the opposite direction than it should).